docs/introduction.md

```diff
@@ -1,13 +1,29 @@
 ---
-id: introduction 
-title: Quick Start
-sidebar_label: Quick Start
+id: introduction
+title: Introduction to Avolites Titan
+sidebar_label: Introduction to Avolites Titan
 ---
 
-This section is a quick guide on how to do most things on Avolites Titan
-consoles. Each section has a link to take you to the appropriate manual
-section for more details.
+Welcome to Avolites Titan! This manual aims to you tell you everything
+there is to know about operating your Avolites console (that's quite a lot).
 
-Softkeys (the A-G menu buttons next to the screen) are shown like 
-\[This\]. Physical labelled buttons on the console are
-shown like \<This\>.
+If you're already familiar with using lighting consoles but new to Titan, 
+the Quick Start section tells you how to do the most common things.
+
+After that we try to take you through all the things you will need to know in the order
+that you're most likely to need them, so we start with how to connect up the console,
+then look at patching lights. After that we go through how to control the different
+attributes of the lights, then how to store frequently-used settings in palettes.
+Titan has some great built-in effects, so we look at those, then how to build a show
+by recording cues, chases and cue lists. Later there are sections on Capture Visualiser, 
+the video integration system Synergy, help when running a show, remote console control,
+and system options. 
+
+Finally we end up with a glossary which covers some of the lighting terms
+which get used in this manual, and also explains some of the terms which mean
+different things on other brands of console.
+
+You can use the search box top right to look for particular terms, or read
```

Comment:
This may be worth omitting for the PDF

Reply:
Is there a way to omit sections for the PDF?

Or could say, "If you are reading the online version of this manual, you can use the search...."

Reply:
It's not so easy to omit particular lines in the PDF.... Also the search and sidebar will move location if you're on a smaller screen/mobile